### PR TITLE
Java sdk

### PR DESCRIPTION
**Pull request by Zhiw**

### CODE REVIEW

File: zh_CN/connect_to_deployments/java_sdk.md [hunk 1]
```diff
@@ -1,7 +1,221 @@
...
+    <version>1.2.5</version>
+</dependency>
+```
+然后运行
+
+```bash
+mvn install
+```
+### 连接
+请在控制台的 [部署概览](../deployments/view_deployment.md) 找到相关的地址以及端口信息，需要注意如果是基础版，端口不是 1883 或 8883 端口，请确认好端口。并且在[认证鉴权](../deployments/auth_and_acl.md) 中添加认证信息。
```
Comment (zibuyu1995) [inline zh_CN/connect_to_deployments/java_sdk.md:32]: @Adek06 @wuzufeng  连接这里按照这个来说明，需要提示用户端口不一样

Comment (wuzufeng) [reply zh_CN/connect_to_deployments/java_sdk.md:32]: ok

File: en_US/connect_to_deployments/java_sdk.md [hunk 2]
```diff
@@ -1,7 +1,214 @@
-# Connect to Deployments with JAVA SDK
+# Connect to Deployments with Java SDK
 
-In this tutorial, you will learn how to use **Eclipse Paho Java Client** to connect to EMQ X Cloud deployment.
+In this tutorial, you will learn how to use [**Eclipse Paho Java Client**](https://github.com/eclipse/paho.mqtt.java) to connect to EMQ X Cloud deployment.
```
Comment (wuzufeng) [inline en_US/connect_to_deployments/java_sdk.md:3]: @azuregaya 各语言英文文档相似部分翻译需要统一一下，可以出个英文的模板

File: zh_CN/connect_to_deployments/java_sdk.md (hunk 1 above)
Comment (Adek06) [reply zh_CN/connect_to_deployments/java_sdk.md:32]: 已修改

**Review by zibuyu1995 — APPROVED** [review]

File: en_US/connect_to_deployments/java_sdk.md (hunk 2 above)
Comment (azuregaya) [reply en_US/connect_to_deployments/java_sdk.md:3]: 好的，我整理下发出来

File: en_US/connect_to_deployments/java_sdk.md [hunk 3]
```diff
@@ -1,7 +1,214 @@
...
+    <version>1.2.5</version>
+</dependency>
+```
+Run
+
+```bash
+mvn install
+```
+### Connect
+You can view the information of connection on  [Deployment Overview](../deployments/view_deployment.md). Please note that the port is not 1883 or 8883  if your edition is not dedicated, make sure you get the right port. And you must add the authentication on [Authentication & ACL](../deployments/auth_and_acl.md).
```
Comment (azuregaya) [inline en_US/connect_to_deployments/java_sdk.md:28]: Please note that the port is not 1883 or 8883  if you choose a Basic package, make sure you get the right port. And you must add the authentication on [Authentication & ACL](../deployments/auth_and_acl.md) in advance.

File: en_US/connect_to_deployments/java_sdk.md [hunk 4]
```diff
@@ -1,7 +1,214 @@
...
+Message published
+Received message: 
+  topic：test/topic
+  Qos：1
+  payload：Hello World
+deliveryComplete
+Disconnected
+```
+
+We can see that we have connected, subscribed and published, and received the published message.
```
Comment (azuregaya) [inline en_US/connect_to_deployments/java_sdk.md:211]: That's the whole process of connecting, subscribing, publishing, and receiving the message.

**Comment by Zhiw** [comment]
@azuregaya 已修改

**Review by wuzufeng — APPROVED** [review]

**Review by chenx-work — APPROVED** [review]

**Review by Adek06 — APPROVED** [review]

**Pull request merged**